backend/tests/test.cpp

```diff
@@ -0,0 +1,29 @@
+#define CATCH_CONFIG_MAIN
+#include <regex>
+#include <catch2/catch_all.hpp>
```

Comment:
**suggestion**: You can use `#include <catch2/catch_test_macros.hpp>` instead of `catch_all` to reduce the size of the import and thus compilation time.